activedirectory/azure-ad-integration-spring-boot-autoconfigure/src/main/java/com/microsoft/azure/autoconfigure/aad/UserPrincipal.java

```diff
@@ -155,21 +157,21 @@ public void verify(JWTClaimsSet claimsSet, SecurityContext ctx) throws BadJWTExc
 
     private List<UserGroup> loadUserGroups(String graphApiToken) throws Exception {
         final String responseInJson = AzureADGraphClient.getUserMembershipsV1(graphApiToken);
-        final List<UserGroup> userGroups = new ArrayList<UserGroup>();
+        final List<UserGroup> lUserGroups = new ArrayList<>();
```

Comment:
could you keep the previous coding style?

if changed to `lUserGroups`, a lot of other variable names have to be changed.

Reply:
There is a class level variable userGroups and the same name is being used locally inside this method which is causing readability issue and to distinguish both variables I have changed this. If i revert back there can u suggest name for class level variable?